src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,43 @@
 import React from 'react';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+
+import { Post } from '../../types/post';
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({
+  post: {
+    title,
+    body,
+    user,
+    comments,
+  },
+}) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        {user ? <UserInfo user={user} /> : ''}
```

Comment:
```suggestion
        {user && <UserInfo user={user} />}
```

Reply:
use logical operator instead of ternary with empty string

Reply:
ok